src/Misc/layoutroot/run-helper.sh.template

```diff
@@ -44,17 +28,17 @@ elif [[ $returnCode == 1 ]]; then
     exit 0
 elif [[ $returnCode == 2 ]]; then
     echo "Runner listener exit with retryable error, re-launch runner in 5 seconds."
-    safe_sleep
+    ./bin/safe_sleep.sh 5
```

Comment:
I'm always a bit iffy on relative paths, does is work fine if I start run.sh with an absolute path from a different directory?

Reply:
i changed to absolute paths and changed the location of the `safe_sleep` script, so it is not in bin folder.